src/Spout/Writer/ODS/Internal/Worksheet.php

```diff
@@ -191,27 +192,33 @@ protected function getCellXML($cellValue, $styleIndex, $numTimesValueRepeated)
             $data .= ' table:number-columns-repeated="' . $numTimesValueRepeated . '"';
         }
 
-        if (CellHelper::isNonEmptyString($cellValue)) {
+        if ($cellValue instanceof Cell) {
```

Comment:
Can this be moved in the calling function to avoid duplication across all writers?